loadflow/loadflow-validation/src/main/java/com/powsybl/loadflow/validation/ValidationTool.java

```diff
@@ -86,7 +86,7 @@ public Options getOptions() {
                     .argName("VALIDATION_WRITER")
                     .build());
             options.addOption(Option.builder().longOpt(TYPES)
-                    .desc("validation types (FLOWS/GENERATORS/BUSES/SVCS) to run, all of them if the option if not specified")
+                    .desc("validation types (FLOWS/GENERATORS/BUSES/SVCS/SHUNTS) to run, all of them if the option if not specified")
```

Comment:
You should use the ValidationType values to generate this list as it’s done in the other tools